src/main/java/uk/gov/hmcts/dts/fact/controllers/SearchController.java

```diff
@@ -59,6 +64,17 @@ public SearchController(final CourtService courtService) {
         }
     }
 
+    @GetMapping(path = "/results/{postcode}")
+    @ApiOperation("Find closest courts by postcode")
+    @Description("Endpoint to return the 10 closest courts for a provided postcode")
+    @SuppressWarnings("PMD.UseObjectForClearerAPI")
+    public ResponseEntity<List<CourtReferenceWithDistance>> findCourtsByPostcode(
+        @Pattern(regexp = "^[a-z]{1,2}\\d[a-z\\d]?\\s*\\d[a-z]{2}$",
```

Comment:
What if some of the postcode letters are in upper case? Is that not going to be blocked by the validation stuff?

Reply:
The frontend lower cases the postcode before it's sent in, which then matches the above. So we shouldn't get upper case ones in. Interesting though! Maybe we should cater for both on the above incase its reused elsewhere in the future @KianKwa

Reply:
@amstevenson Think we should treat the API endpoints as independent from the front end usage. Might be better to cater for upper case too if you agree with that

Reply:
Discussed with Kian and this can be resolved